sw/host/provisioning/ft/src/main.rs

```diff
@@ -2,15 +2,20 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
+use std::collections::HashMap;
 use std::path::PathBuf;
 use std::time::Duration;
 
-use anyhow::{bail, Result};
+use anyhow::{bail, Context, Result};
 use clap::{Args, Parser};
+use elliptic_curve::pkcs8::DecodePrivateKey;
+use elliptic_curve::SecretKey;
+use p256::NistP256;
+use serde_json::from_str;
```

Comment:
You can eliminate the `use` statement if you refer to the full path `serde_json::from_str` at the call site.

Reply:
fixed.